vars/production.yml

```diff
@@ -15,10 +15,10 @@ router:
 
 api:
   memory: 2GB
-  instances: 20
+  instances: 10
 
 buyer-frontend:
-  instances: 10
+  instances: 8
```

Comment:
I thought we were on the default (5) for buyer before? Can we just delete this bit?

Reply:
If ya like